CvGameCoreDLL_Expansion2/Lua/CvLuaPlayer.cpp

```diff
@@ -1473,6 +1473,7 @@ static DomainTypes LuaToTradeDomain(lua_State* L, int index)
 		return eDomain;
 	default:
 		luaL_error(L, "Invalid trade domain index %d", iDomain);
+		UNREACHABLE();
```

Comment:
mark luaL_error noreturn?

Reply:
Looks like these functions are documented as never returning so I see no harm in doing so. I'll change the function definitions as appropriate.